### PR TITLE
GitOps | Changing test profile for the remaining ci-operator configs

### PR DESCRIPTION
**Pull request by varshab1210**

### CODE REVIEW

**Comment by varshab1210** [comment]
/pj-rehearse pull-ci-redhat-developer-gitops-operator-v1.9-v4.13-kuttl-parallel

**Comment by varshab1210** [comment]
We can ignore the test failure, the new test profile works `--- FAIL: kuttl/harness/1-073_validate_rhsso`

/pj-rehearse ack

File: ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.9-presubmits.yaml [hunk 1]
```diff
@@ -298,12 +303,18 @@ presubmits:
           readOnly: true
       serviceAccountName: ci-operator
       volumes:
+      - name: boskos
+        secret:
+          items:
+          - key: credentials
+            path: credentials
+          secretName: boskos-credentials
```
Comment (anandrkskd) [inline ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.9-presubmits.yaml:311]: what is this `boskos` creds that we are adding?

Comment (varshab1210) [reply ci-operator/jobs/redhat-developer/gitops-operator/redhat-developer-gitops-operator-v1.9-presubmits.yaml:311]: This was added automatically, must be related to the test-profile change

**Comment by anandrkskd** [comment]
Just one questioned mentioned above, other than that 
/lgtm

**Pull request merged**